src/platf/libplatfplaceholder.cpp

```diff
@@ -0,0 +1,12 @@
+#include "libdisplaydevice/libplatfplaceholder.h"
+
+#ifdef DD_WIN
```

Comment:
```suggestion
#ifdef _WIN32
```

Should we do this instead, as it's more universal?

Reply:
I will remove it as it's just a proof-of-concept for now, but I don't think it that simple:
https://stackoverflow.com/questions/142508/how-do-i-check-os-with-a-preprocessor-directive

Funnily enough, DD_WIN seems to be the simpler approach